doc/admin-guide/files/ssl_server_name.config.en.rst

```diff
@@ -0,0 +1,138 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+   or more contributor license agreements.  See the NOTICE file
+   distributed with this work for additional information
+   regarding copyright ownership.  The ASF licenses this file
+   to you under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License.
+
+.. include:: ../../common.defs
+
+ssl_server_name.config
+**********************
+
+.. configfile:: ssl_server_name.config
+
+Description
+===========
+
+This file is used to configure aspects of TLS connection handling for both inbound and outbound
+connections. The configuration is driven by the SNI values provided by the inbound connection. The
+file consists of a set of configuration items, each identified by an SNI value (``fqdn``).
+When an inbound TLS connection is made, the SNI value from the TLS negotiation is matched against
+the items specified by this file and if there is a match, the values specified in that item override
+the defaults. This is done during the inbound connection processing and be some outbound properties
+can be overridden again later, such as via :file:`remap.config` or plugins.
+
+By default this is named :file:`ssl_server_name.config`. The file can be changed by settting
+:ts:cv:`proxy.config.ssl.servername.filename`.
+
+The configuration file is Lua based. After parsing and executing the Lua code, the global Lua
+variable :code:`server_config` is examined. Its value should be a list of tables. Each table is set
+of key / value pairs that create a configuration item.
+
+This file is loaded on start up and by :option:`traffic_ctl config reload` if the file has been modified
+since process start.
+
+======================= ==============================================================================
+Key                     Meaning
+======================= ==============================================================================
+fqdn                    Fully Qualified Domain Name. This item is used if the SNI value matches this.
```

Comment:
the fonts look mixed up in this table. is this normal?

Reply:
Fixed. It's a style sheet problem, where the styling is different depending on whether there is 1 paragraph or multiple paragraphs in the cell (in the former, it's just text, but in the latter there are actual `p` tags). I put in a work around.